core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/helpers/impl/LinkHelperImpl.java

```diff
@@ -131,12 +131,6 @@ private boolean updateValueMapWithNewLink(ModifiableValueMap modifiableValueMap,
     }
 
     private Object updatePropertyWithNewLink(Object value, String currentLink, String newLink) {
-        return getLinkStream(value)
-                .map(Result::getValue)
-                .filter(currentLink::equals)
-                .findFirst()
-                .map(currentLinkToReplace ->
-                        LinkInspectorResourceUtil.replaceStringInPropValue(value, currentLinkToReplace, newLink))
-                .orElse(null);
+        return LinkInspectorResourceUtil.replaceStringInPropValue(value, currentLink, newLink);
```

Comment:
Consider removing the `updatePropertyWithNewLink` method. Just call `LinkInspectorResourceUtil.replaceStringInPropValue` wherever you call `updatePropertyWithNewLink`. There is no sense in a method that just calls another method, and that's all